examples/vpc_sc_perimeter_auto_add/README.md

```diff
@@ -0,0 +1,25 @@
+# Auto-add project to VPC Service Controls perimeter
+
+This is a demo of adding a newly-created project to a [VPC Service Controls perimeter](https://cloud.google.com/vpc-service-controls/).  Currently, there's not a Google Cloud product-level feature that automatically adds a new project to a service perimeter; so we have to rely on automation to do this.
+
+## Prerequisites
+To complete this demo, you'll need:
+- A Google Cloud [organization](https://cloud.google.com/resource-manager/docs/quickstart-organizations) mapped to a domain that you own with at least one project already created.
+- A valid billing account.
+- A service account with "Access Context Manager Admin" privileges.  
+- Terraform v0.11.13 with provider.google v2.6.0 (the only versions I've tested; run `terraform version` to see what you've got).
+
```

Comment:
You can add the **Running the example** section here.
Here's a draft that's partly consistent with the other examples.

```suggestion
## Running the example

To provision this example, run the following from within this directory:
- `terraform init` to get the plugins
- `terraform plan` to see the infrastructure plan
- `terraform apply` to apply the infrastructure build
- `terraform destroy` to destroy the built infrastructure

### Cleaning up
- When you're done with this demo and have successfully deleted all the resources with `terraform destroy`, delete the service account keys stored locally and [delete the service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts#deleting_a_service_account) altogether from the Cloud Console.
```

Reply:
Removed vpc_sc_perimeter_auto_add from this pull request, no longer relevant.